resources/sass/pages/_cart.scss

```diff
@@ -0,0 +1,212 @@
+.flex {
+    @include flex;
+}
+
+.flex-row {
+    @include flex-row;
+}
+
+.flex-column {
+    @include flex-column;
+}
+
+.flex-row-reverse {
+    @include flex;
+    flex: 1;
+    flex-direction: row-reverse;
+}
+
+.flex-column-reverse {
+    @include flex;
+    flex: 1;
+    flex-direction: column-reverse;
+}
+
+main.cart {
+    @extend .flex-column;
+    background: #eeeeee;
+    padding: 33px 0px;
+    max-width: 1140px;
+    margin: 0 auto;
+    position: relative;
+    .cart-header {
+        @extend .flex-column;
+        background: $color_black;
+        padding: 10px;
+        &__text {
+            color: $color_white;
+        }
+    }
+}
+
+.cart-item {
+    @extend .flex-row;
+    margin-top: 10px;
+    background: $color_white;
+    &__logo {
+        width: 20%;
+        height: 20%;
+    }
+}
+
+.cart-item-detail {
+    @extend .flex-column;
+    &__heading {
+        font-size: 1.5rem;
+        padding: 15px;
+    }
+    &__info {
+        @extend .flex-row;
+        font-size: 1rem;
+        align-items: center;
+        &--count {
+            @extend .flex-row;
+        }
+        &--remove,
+        &--add {
```

Comment:
Where did you copy this code from, this exact module was there in Navin's PR too. @ankurkakroo2